content/6_Advanced/BCC_2CC.mdx

```diff
@@ -25,7 +25,73 @@ frequency: 1
 
 <Problems problems="sam2" />
 
-(implementation)
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+const int NMAX = 3e5;
+
+int n, m, timer, SCC;
```

Comment:
declare variables where you actually use them

Reply:
Done.